source/_integrations/sighthound.markdown

```diff
@@ -63,3 +63,45 @@ source:
       required: false
       type: string
 {% endconfiguration %}
+
+To verify the integration, check if a new entity is appeared as `image_processing.sighthound_my_cam`
+
+## Process an Image
+
+When you want to process an image, you have to call `image_processing.scan` service and listen to the sighthound.person_detected and/or sighthound.vehicle_detected events.
+
+A simple example is via 2 two automations:
```

Comment:
"Simple" is subjective, and can be a bit problematic for users that don't feel like it is simple.

```suggestion
An example using two automations:
```